demisto_sdk/commands/upload/uploader.py

```diff
@@ -421,6 +428,29 @@ def print_summary(self) -> None:
 
             logger.info(f"[green]SUCCESSFUL UPLOADS:\n{uploaded_str}\n[/green]")
 
+        if self._skipped_upload_marketplace_mismatch:
```

Comment:
can you show an example of an output here (attach screenshot to PR's description) in cases where you have some content-items that are supported and some that are not.